app/main/views/briefs.py

```diff
@@ -116,10 +116,107 @@ def start_brief_response(brief_id):
     )
 
 
-@main.route('/opportunities/responses/<int:brief_response_id>/edit', methods=['GET'])
+@main.route('/opportunities/<int:brief_id>/responses/<int:brief_response_id>', methods=['GET'])
+@main.route(
+    '/opportunities/<int:brief_id>/responses/<int:brief_response_id>/<string:section_id>',
+    methods=['GET', 'POST']
+)
+@feature.is_active_feature('NEW_SUPPLIER_FLOW')
 @login_required
-def edit_brief_response(brief_response_id):
-    return 'Hello world'
+def edit_brief_response(brief_id, brief_response_id, section_id=None):
+    brief = get_brief(data_api_client, brief_id, allowed_statuses=['live'])
+    brief_response = data_api_client.get_brief_response(brief_response_id)['briefResponses']
+
+    if brief_response['briefId'] != brief['id'] or brief_response['supplierId'] != current_user.supplier_id:
+        abort(404)
+
+    if not is_supplier_eligible_for_brief(data_api_client, current_user.supplier_id, brief):
+        return _render_not_eligible_for_brief_error_page(brief)
+
+    if supplier_has_a_brief_response(data_api_client, current_user.supplier_id, brief_id):
+        flash('already_applied', 'error')
+        return redirect(url_for(".view_response_result", brief_id=brief_id))
+
+    framework, lot = get_framework_and_lot(
+        data_api_client, brief['frameworkSlug'], brief['lotSlug'], allowed_statuses=['live'])
+
+    max_day_rate = None
+    role = brief.get('specialistRole')
+    if role:
+        brief_service = data_api_client.find_services(
+            supplier_id=current_user.supplier_id,
+            framework=brief['frameworkSlug'],
+            status="published",
+            lot=brief["lotSlug"],
+        )["services"][0]
+        max_day_rate = brief_service.get(role + "PriceMax")
+
+    content = content_loader.get_manifest(
```

Comment:
@tombye `max_day_rate` is needed to get the correct content. `filter` is called on the content object on line 156 and `max_day_rate` is passed in. It's to do with the way the new jinja stuff works. Any variable you want to be available to jinja get's passed in here. It gets consumed [here](https://github.com/alphagov/digitalmarketplace-frameworks/blob/ess-req-met/frameworks/digital-outcomes-and-specialists/questions/brief-responses/dayRate.yml#L3).